### PR TITLE
Alyssa/i os/6.5.1

### PR DESCRIPTION
**Pull request by alyssa-embrace**
Upgrading pipeline and plugin for 6.5.2

### CODE REVIEW

**Comment by alyssa-embrace** [comment]
Pending some more testing currently.

File: setup_iOS_build_environment.sh [hunk 1]
```diff
@@ -1,18 +1,19 @@
 #!/bin/bash
 
+# Set the release version to download, default to current supported if not provided
+RELEASE_VERSION=${1:-6.5.2}
```
Comment (jpmunz) [inline setup_iOS_build_environment.sh:4]: should this be 6.5.1?

File: Embrace Unity iOS Interface/Sources/EmbraceManager.swift [hunk 2]
```diff
@@ -46,7 +46,11 @@ public class EmbraceManager: NSObject {
             
             return true
         } catch let e {
-            os_log("Error starting Native Embrace SDK \(e.localizedDescription)")
```
Comment (jpmunz) [inline Embrace Unity iOS Interface/Sources/EmbraceManager.swift:49]: curious if these changes were prompted by a customer issue? We do the same `os_log` call in the React Native SDK, should we have a similar check there?

File: setup_iOS_build_environment.sh (hunk 1 above)
Comment (alyssa-embrace) [reply setup_iOS_build_environment.sh:4]: This should be 6.5.2 actually; we've shifted the target I think and I need to rename the branch.

File: Embrace Unity iOS Interface/Sources/EmbraceManager.swift (hunk 2 above)
Comment (ArielDemarco) [reply Embrace Unity iOS Interface/Sources/EmbraceManager.swift:49]: I've changed the SDK to have iOS 13 as the `MINIMUM_DEPLOYMENT_TARGET` ; that means that `os_log` APIs (at least the ones that we were using here) are only available if the OS is `14.0` or above.

Comment (alyssa-embrace) [reply Embrace Unity iOS Interface/Sources/EmbraceManager.swift:49]: Did you change that in the xcodegen `project.yml` file or the original `.xcodeproj`?

Comment (ArielDemarco) [reply Embrace Unity iOS Interface/Sources/EmbraceManager.swift:49]: The original `.xcodeproj`: (aka `Embrace Unity iOS Interface/Embrace Unity iOS Interface.xcodeproj/project.pbxproj`)

Comment (jpmunz) [reply Embrace Unity iOS Interface/Sources/EmbraceManager.swift:49]: got it, ya we only specify 13 as our minimum in React Native, ok we should make this change there as well

**Review by jpmunz — APPROVED** [review]

**Review by ArielDemarco — COMMENTED** [review]
LGTM

**Pull request merged**